src/Illuminate/Database/Query/Builder.php

```diff
@@ -1256,6 +1256,165 @@ public function whereNotNull($columns, $boolean = 'and')
         return $this->whereNull($columns, $boolean, true);
     }
 
+    /**
+     * Add a where LIKE 'expression%' statement to the query.
+     *
+     * @param  string  $column
+     * @param  mixed  $value
+     * @param  string  $boolean
+     * @param  bool  $not
+     * @return $this
+     */
+    public function whereStartsWith($column, $value, $boolean = 'and', $not = false)
+    {
+        return $not === false
+            ? $this->where($column, 'LIKE', "{$value}%", $boolean)
```

Comment:
This is incorrect. You need to escape the value otherwise input like `_`, `%` and `\` will have special meaning... but it's not supposed to.

Reply:
Yes, you are correct. I will consider the changes.